spec/recurly/item_spec.rb

```diff
@@ -57,9 +92,9 @@
       it "sends changed attributes to the server" do
         stub_api_request :get, 'items/plastic_gloves', 'items/show-200'
         stub_request(:put, "https://api.recurly.com/v2/items/plastic_gloves").
-        with(:body => "<item><name>Sleek Plastic Gloves</name></item>",
+        with(:body => "<item><name>Sleek Plastic Gloves</name><revenue_gl_account_id>aksdfu48</revenue_gl_account_id></item>",
           :headers => Recurly::API.headers).to_return(:status => 200, :body => "", :headers => {})
-        item.update_attributes({ name: 'Sleek Plastic Gloves' })
+        item.update_attributes({ name: 'Sleek Plastic Gloves', revenue_gl_account_id: 'aksdfu48' })
```

Comment:
would be nice to add the other RevRec attributes we can update liability_gl_account_id and performance_obligation_id.

Reply:
Done!